src/modules/aws_ecs_fargate@0.0.1/entity/container_definition.ts

```diff
@@ -31,7 +31,11 @@ export class ContainerDefinition {
   @Column()
   name: string;
 
-  @ManyToOne(() => TaskDefinition)
+  @ManyToOne(() => TaskDefinition, {
+    // if the parent task def is deleted, also delete this container def
```

Comment:
Comment seems slightly out of sync with the code here?

Reply:
typeorm is a bit funny in that you define this in the child as opposed to the parent which took me some time to realize :/

Reply:
So I mean you're also doing it if the parent task definition is updated, afaict?

Reply:
ooh yes, that part of the comment is incomplete. I see what you mean, will update in the next PR